pysam/libcalignedsegment.pxd

```diff
@@ -57,15 +57,15 @@ cdef class AlignedSegment:
     cpdef has_tag(self, tag)
 
     # returns a valid sam alignment string
-    cpdef tostring(self, AlignmentFile_t handle)
+    cpdef tostring(self, htsfile=*)
```

Comment:
Not clear why this method needs to be `cpdef`.  Do we call it often from a `cdef` function?